### PR TITLE
Add survey banner to footer

### PR DESCRIPTION
**Pull request by belbiy**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [X] Bug fix or improvement

## Summary

When this pull request is merged, it will add a survey banner to the Cloud section.

## Additional information

List all affected URLs 

- cloud/*

### CODE REVIEW

**Comment by belbiy** [comment]
running tests

File: common/js/survey.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  // Fist, check if browser supports storing the data
+  if (typeof localStorage !== 'undefined') {
+
+    var surveyName = "Cloud survey 2 done";
+
+    // Only show if user didn't interact with the banner
+    if ( !localStorage.getItem(surveyName) ) {
+
+      // Build banner text
+      var $banner = $('<div class="bs-callout" style="display:none; text-align:left; max-width: 1024px; margin-top: 50px;"><h2>Hello Magento Community!</h2><p>We at Magento believe in listening to our community, as they play an important role in building valuable products in the future!  If you want to make an impact on Magento Cloud please share your feedback through this survey.</p><p>Thank you for advance participating in our survey. Your feedback is 100% confidential and will not be shared outside of Magento.</p> <button data-href="https://www.surveymonkey.com/r/CloudFeatures_LP2" class="btn btn-primary btn-large">OK, I will help</button> <button class="btn btn-large">No, thanks</button> </div>');
```
Comment (meker12) [inline common/js/survey.js]:
```suggestion
      var $banner = $('<div class="bs-callout" style="display:none; text-align:left; max-width: 1024px; margin-top: 50px;"><h2>Hello Magento Community!</h2><p>We at Magento believe in listening to our community, as they play an important role in building valuable products in the future!  If you want to make an impact on Magento Cloud please share your feedback through this survey.</p><p>Thank you in advance for participating in our survey. Your feedback is 100% confidential and will not be shared outside of Magento.</p> <button data-href="https://www.surveymonkey.com/r/CloudFeatures_LP2" class="btn btn-primary btn-large">OK, I will help</button> <button class="btn btn-large">No, thanks</button> </div>');
```

Comment (belbiy) [reply common/js/survey.js]: Thank you, i'll made the text of the banner little bit easier to read next time!

**Comment by belbiy** [comment]
Here's the survey banner deployed in staging https://devdocs.magedevteam.com/110/guides/v2.3/cloud/bk-cloud.html

**Comment by belbiy** [comment]
running tests

**Pull request merged**